src/FSAL/FSAL_CORTXFS/conf/nfs_setup.sh

```diff
@@ -303,6 +297,65 @@ function cortx_nfs_config {
 	echo -e "\nNFS setup is complete"
 }
 
+function cortx_ganesha_restart {
+	echo -e "\nStopping NFS Ganesha"
+	systemctl stop nfs-ganesha || die "Failed to stop NFS-Ganesha"
```

Comment:
Just a suggestion: will it be good, if we check first whether Ganesha -service is running then stop it?

Reply:
> Just a suggestion: will it be good, if we check first whether Ganesha -service is running then stop it?

We can do that, but the script works fine and does its job if the service is already stopped. Let me know if that is needed.